sky/packages/sky/lib/sky_tool

```diff
@@ -447,6 +488,27 @@ class AndroidDevice(object):
         logging.info(' '.join(cmd))
         subprocess.check_output(cmd)
 
+    def logs(self):
+        def do_logs():
+            cmd = [
+                self.adb_path,
+                'logcat',
+                '-s',
+                'sky',
+                'chromium',
```

Comment:
Let me check -- I pulled this invocation out of the docs, but it could be wrong.